File: PHP/Ram.php
<?php
class Ram{
    //atribut
    private $capacity;
    private $price;

    //constructor
    function __construct($capacity, $price) {
        $this->capacity=$capacity;
        $this->price=$price;
    }

    //Setter Getter
    function setCapacity($capacity) { $this->capacity=$capacity; }
    function setPrice($price) { $this->price=$price; }

    function getCapacity() { return $this->capacity; }
    function getPrice() { return $this->price; }

    //Method
    //Menampilkan atribut Ram
    function printRam() {
        echo "RAM Capacity: " . $this->getCapacity() . "<br>";
        echo "RAM Price: " . $this->getPrice() . "<br>";
    }

    function __destruct() {}
}

File: C++/RAM.cpp
#include <string>
#include <iostream>
using namespace std;

class RAM {
    private:
        //atribut
        int capacity;
        int price;
    public:
        //constructor
        RAM() {}
        RAM(int capacity, int price) {
            this->capacity=capacity;
            this->price=price;
        }
        //Setter Getter
        void setCapacity(int capacity) { this->capacity=capacity; }
        void setPrice(int price) { this->price=price; }

        int getCapacity() { return this->capacity; }
        int getPrice() { return this->price; }

        //Method
        void printRAM() {
            cout<<"RAM Capacity: "<<this->capacity<<endl;
            cout<<"RAM Price: "<<this->price<<endl;
        }
        //Destructor
        ~RAM() {}
};